### PR TITLE
added branch

### PR DESCRIPTION
**Pull request by AzizRAA**
In this project I recreated Awesome books with ES6 and modules

### CODE REVIEW

File: modules/date.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
```
Comment (1GORDON) [inline modules/date.js:1]: Great job @AzizRAA 🤝

- [ ] As stated in the project requirements you need to add the [Luxon](https://moment.github.io/luxon/#/install?id=es6) and import it into the `index.js` file. For your case, you may import it to the `date.js`. Make sure it displays below the navigation bar.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    this.title = title;
+    this.author = author;
+  }
+
+  saveBook() {
+    const savedBook = {
+      title: this.title,
+      author: this.author,
+    };
+
```
Comment (1GORDON) [inline index.js:24]: Great job @AzizRAA 🤝

- [ ] As stated in the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md). Kindly use ES6 syntax for all your functions and methods. Make sure this `saveBook` is written using ES6 syntax.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                <h3>Awesome Books</h3>
+                <ul>
+                    <li class="link"><a href="">List</a></li>
+                    <li class="link"><a href="">Add new</a></li>
+                    <li class="link"><a href="">Contact</a></li>
+                </ul>
+            </nav>
+            <p class="date"></p>
+            <section class="container"> 
+                <div id="library" class="slide library">
```
Comment (1GORDON) [inline index.html:24]: Great job @AzizRAA 🤝

- [ ] Kindly make sure all sections are displaying properly on your design. As of now, `list`, `add new`, and `contact` are all rendering a blank page.
dispaly

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @AzizRAA ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Kindly refactor your readme file correctly. Remember this is the first document that describes what your project is all about. Add the project name, description, the correct author for this project, and all sections as listed from the [template](https://github.com/microverseinc/readme-template).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/date.js (hunk 1 above)
Comment (AzizRAA) [reply modules/date.js:1]: Thanks yes it displays below the navbar.

File: index.js (hunk 2 above)
Comment (AzizRAA) [reply index.js:24]: sure Thanks

File: index.html (hunk 3 above)
Comment (AzizRAA) [reply index.html:24]: Yes they are displaying correctly I hope you checked modules branch.

**Comment by AzizRAA** [comment]
I will update Readme sorry i forgot

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @AzizRAA, 

Great work on making the changes requested by a previous reviewer 👏 🔥
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 🎉🔥

 # Required Changes ♻️
- [ ] Let's update your readme file as requested by the previous reviewer with the project info starting with the name, etc.

_Check the comments under the review._

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @AzizRAA  ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**